### PR TITLE
set new colors for price label

### PR DESCRIPTION
**Pull request by shayanbinary**

### CODE REVIEW

File: src/store/ChartStore.js [hunk 1]
```diff
@@ -123,6 +123,59 @@ class ChartStore {
...
+            let yaxisLabelStyle = this.yaxisLabelStyle;
+            if (yax.yaxisLabelStyle) yaxisLabelStyle = yax.yaxisLabelStyle;
+            const params = {
+                ctx:context,
+                x,
+                y,
+                top:y - (height / 2),
+                width,
+                height,
+                radius,
```
Comment (shayanbinary) [inline src/store/ChartStore.js]: Referring to the design of `current price label` it shouldn't have a radius, and this prop only can be handled here. that's why I use monkey patching for here.

File: src/store/CurrentSpotStore.js [hunk 2]
```diff
@@ -47,9 +47,7 @@ class CurrectSpotStore {
             }
         }
 
-        this.show = visible && (layout.chartType !== 'candle'
-                && layout.chartType !== 'colored_bar'
-                && layout.chartType !== 'hollow_candle');
+        this.show = visible && (layout.aggregationType !== 'pandf');
```
Comment (shayanbinary) [inline src/store/CurrentSpotStore.js]: As Designers request to show spot on every chart type except `point & figure`.

File: src/store/ChartStore.js [hunk 3]
```diff
@@ -123,6 +123,59 @@ class ChartStore {
         this.chartNode = this.rootNode.querySelector('.ciq-chart-area');
         this.chartControlsNode = this.rootNode.querySelector('.cq-chart-controls');
 
+        CIQ.ChartEngine.prototype.createYAxisLabel = function (panel, txt, y, backgroundColor, color, ctx, yAxis) {
```
Comment (MahboobehMohammadi) [inline src/store/ChartStore.js]: @shayanbinary I dont think its good to override the whole `createYAxisLabel `, if we cant change the style can you check for a listener or injecttion, something like `headsUpHR` and just change float label inline style there?

Comment (shayanbinary) [reply src/store/ChartStore.js]: @MahboobehMohammadi actually by `drawCurrentHR` you just be able to set color and background, but for the new design, we need to handle `radius, height and width` which these props effect directly from this method

**Pull request merged**